modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/src/main/java/com/liferay/headless/admin/list/type/internal/dto/v1_0/util/ListTypeEntryUtil.java

```diff
@@ -58,4 +61,16 @@ public static ListTypeEntry toListTypeEntry(
 		return listTypeEntry;
 	}
 
+	private static Map<Locale, String> _getLabelMap(
+		Map<Locale, String> localizedMap, String name) {
+
+		Locale siteDefaultLocale = LocaleUtil.getSiteDefault();
+
+		if (!localizedMap.containsKey(siteDefaultLocale)) {
+			localizedMap.put(siteDefaultLocale, name);
+		}
+
+		return localizedMap;
+	}
```

Comment:
This code is repeated in this PR. There are two methods, and both are almost the same. Can we simplify and use the method computeIfAbsent()?

Reply:
I agree, the methods are the same and one of them is unnecessary, since we call them on the same labelMap/localizedMap. Once the method was called in the ResourceImpl class, there's nothing to compute in the Util class.
I would delete one of them. I don't know what do we prefer, add the missing locale in the ResourceImpl class or in the Util class.

Reply:
@evibudai, @marcelabc this code was repeated because it was being used for both the label for the ListTypeDefinition and the Entry labels. Maybe we can extract to a common place, but both should be used

Reply:
In the ListTypeDefinitionResource class it was also being used for both post and put methods

Reply:
I sent video proof on Slack that it works like this both for the first time import and the later updates